OrbitLinuxTracing/PerfEventRecords.h

```diff
@@ -5,33 +5,34 @@
 
 namespace LinuxTracing {
 
-// This struct must be in sync with the SAMPLE_TYPE_TID_TIME_CPU in
+// This struct must be in sync with the SAMPLE_TYPE_TID_TIME_STREAMID_CPU in
 // PerfEventOpen.h, as the bits set in perf_event_attr::sample_type determine
 // the fields this struct should have.
-struct __attribute__((__packed__)) perf_event_sample_id_tid_time_cpu {
-  uint32_t pid, tid; /* if PERF_SAMPLE_TID */
-  uint64_t time;     /* if PERF_SAMPLE_TIME */
-  uint32_t cpu, res; /* if PERF_SAMPLE_CPU */
+struct __attribute__((__packed__)) perf_event_sample_id_tid_time_streamid_cpu {
```

Comment:
Consider calling this something like "perf_event_base" or "perf_event_generic".

Reply:
The name would be "perf_event_sample_id", see the other comment for why it isn't.